server/src/main/resources/org/elasticsearch/bootstrap/test-framework.policy

```diff
@@ -104,4 +104,9 @@ grant codeBase "file:${gradle.worker.jar}" {
 grant {
   // since the gradle test worker jar is on the test classpath, our tests should be able to read it
   permission java.io.FilePermission "${gradle.worker.jar}", "read";
-};
\ No newline at end of file
+};
+
+grant codeBase "${codebase.junit-rt.jar}" {
```

Comment:
nit: please move this up so that the bare grant remains the last entry in the file

Reply:
fixed in 7d4e608